src/components/PageSections/SearchBar/_SearchBar.scss

```diff
@@ -0,0 +1,88 @@
+.coa-SearchBar {
+  display: inline-block;
+  width: 43%;
+  text-align: right;
+  padding-top: 11px;
+}
+
+.coa-SearchBar__container {
+  background-color: none;
+  border-radius: $coa-border-radius-medium;
+  height: 40px;
+  padding: 3px;
+  &.open {
+    background-color: $coa-color-white;
+  }
+  @include mobile {
+    display: none;
+  }
+}
+
+.coa-SearchBar__input {
+  display: inline-block;
+  background-color: inherit;
+  border: none;
+  height: 34px;
+  padding-left: 15px;
+  width: 363px;
+  vertical-align: top;
+  margin-right: 7px;
+  border-radius: $coa-border-radius-medium;
+  pointer-events: none;
+  &.open {
+    pointer-events: all;
+  }
+  @media screen and (max-width: 1135px) {
+    width: 300px;
+  }
+  @media screen and (max-width: 1000px) {
```

Comment:
mixins

Reply:
These are actually really unique break points for the search bar in the header. Not equal to any of our mixins. Very much a custom job.